website/docs/r/logs_pipeline.html.markdown

```diff
@@ -0,0 +1,254 @@
+---
+layout: "datadog"
+page_title: "Datadog: datadog_logs_pipeline"
+sidebar_current: "docs-datadog-resource-logs-pipeline"
+description: |-
+  Provides a Datadog logs pipeline resource. This can be used to create and manage logs pipelines.
+---
+
+# datadog_logs_pipeline
+
+Provides a Datadog [Logs Pipeline API](https://docs.datadoghq.com/api/?lang=python#logs-pipelines) resource. This can be used to create and manage Datadog logs pipelines.
+
+
+## Example Usage: Datadog logs pipeline
+
+```hcl
+# Create a new Datadog logs pipeline
+resource "datadog_logs_pipeline" "sample_pipeline" {
+    name = "sample pipeline"
+    is_enabled = true
+    filter {
+        query = "source:foo"
+    }
+    processor {
```

Comment:
Why can't we have a `processors` list containing all the different processors ? It's a bit redundant to type processor all the time, WDYT ?

Reply:
it's a [recommended](https://www.terraform.io/docs/configuration/attr-as-blocks.html#defining-a-fixed-object-collection-value) way of defining list of objects in terraform.